### ISSUE TITLE
private _event and _eventsCount properties are stringifying after a set operation (Node 5.x.x)

### ISSUE THREAD

**Issue by AlmirKadric** [Wizcorp/node-tomes#44]
Take the following sample to see what's happening when using tomes in Node v5.2.0
```
> var tomes = require('./tomes');
undefined
> var tTest = tomes.Tome.conjure('{ "hello": 1}');
undefined
> tTest.on('readable', function () {});
{ [String: '{ "hello": 1}'] _eventsCount: 1 }
> JSON.stringify(tTest, null, '\t');
'"{ \\"hello\\": 1}"'
> tTest.set('hello', []);
ObjectTome { hello: ArrayTome {} }
> JSON.stringify(tTest, null, '\t');
'{\n\t"hello": [],\n\t"_events": {}\n}'
> tTest.on('readable', function () {});
ObjectTome { hello: ArrayTome {} }
> JSON.stringify(tTest, null, '\t');
'{\n\t"hello": [],\n\t"_events": {\n\t\t"readable": [\n\t\t\tnull,\n\t\t\tnull\n\t\t]\n\t}\n}'
```

**Comment by AlmirKadric** [comment]
sample without output
```
var tomes = require('./tomes');
var tTest = tomes.Tome.conjure('{ "hello": 1}');
tTest.on('readable', function () {});
JSON.stringify(tTest, null, '\t');
tTest.set('hello', []);
JSON.stringify(tTest, null, '\t');
tTest.on('readable', function () {});
JSON.stringify(tTest, null, '\t');
```

**Comment by AlmirKadric** [comment]
also tested other node versions:
 - 5.3.0 (problem exists)
 - 4.2.3 (problem exists, but sample needs to change a bit, `require('tomes')` instead)
 - 0.12.9 (problem exists, but sample needs to change a bit, `require('tomes')` instead)
 - 0.10.40 (problem DOES NOT exist)

**Comment by ronkorving** [comment]
My guess is that the EventEmitter implementation changed over Node versions. The problem may be that we don't set a value to _events, and these lines mess things up for us: https://github.com/nodejs/node/blob/0ec093cd410b8797b02055a445d650f72fd16796/lib/events.js#L47-L50

Possibly, if we initialize _events as an empty object, that if-statement won't trigger and the problem may go away.

**Comment by ronkorving** [comment]
Confusingly, you are conjuring a tome from what looks like an object, but is really a string. I'll continue to investigate, but this is .. confusing for a test.

**Comment by ronkorving** [comment]
Wow... so just to confirm here in writing. This problem *only* occurs when we *pretty* stringify our JSON. Simply calling JSON.stringify doesn't expose _events, but when you add `, null, '\t'` arguments, _events shows up. Mind. Blown.

**Comment by ronkorving** [comment]
Fixed by #53
